crates/wasi-common/tokio/src/sched/windows.rs

```diff
@@ -0,0 +1,248 @@
+use crate::asyncify;
+use anyhow::Context;
+use std::ops::Deref;
+use std::os::windows::io::{AsRawHandle, RawHandle};
+use std::sync::mpsc::{self, Receiver, RecvTimeoutError, Sender, TryRecvError};
+use std::sync::Mutex;
+use std::thread;
+use std::time::Duration;
+use wasi_common::{
+    file::WasiFile,
+    sched::{
+        subscription::{RwEventFlags, Subscription},
+        Poll,
+    },
+    Error, ErrorExt,
+};
+
+pub async fn poll_oneoff<'a>(poll: &mut Poll<'a>) -> Result<(), Error> {
+    asyncify(move || poll_oneoff_(poll))
```

Comment:
I'm a little confused by this, how come there's an `asyncify` of an `async` function?

Although reading the above `asyncify` again I kind of understand, but not really.

Reply:
`block_on_dummy_executor`?